client/src/api/okteto.ts

```diff
@@ -100,10 +100,26 @@ const up = (manifestFile: string, contextName: string, onOutputChange: (stdout:
   });
 };
 
+const status = (manifestFile: string, contextName: string, onStatusChange: (status: string) => void) : ExecProcess | undefined => {
+  const args = ['status', '-f', manifestFile, '-c', contextName, '--watch']
+  return window.ddClient.extension?.host?.cli.exec('okteto', args, {
+    stream: {
```

Comment:
@leomindez the output of the `status` command does not stream output, it just returns a single response so you can use the `exec` call without the `stream` options, check other simple calls like `context list` here: https://github.com/okteto/docker-desktop-extension/blob/main/client/src/api/okteto.ts#L40

Remember that the call will return an object with the form: `{"status": "activating"}`. You could return the status directly and based on a defined type, instead of just a string.

Reply:
@leomindez Looking good!!! I just add a few minor comments